router/config/config.js

```diff
@@ -56,4 +56,13 @@ config.UNKNOWN_OPTION_FREEFLOW               = 'unknown_option_freeFlow'
 config.UNKNOWN_OPTION                        = 'unknown_option'
 
 config.TELEGRAM_BOT_ENDPOINT         = 'https://api.telegram.org/bot<bot_secret_key>/sendMessage'
+
+config.ENV_URL = {
+    'diksha.gov.in': 'https://ntpproductionall.blob.core.windows.net/',
```

Comment:
We should not add all environments configurations here. Based on the environment, the URL will be given